docs/README.md

```diff
@@ -38,7 +38,7 @@ installed. Also install the following libraries:
 ```sh
 $ sudo gem install jekyll jekyll-redirect-from rouge
 # Following is needed only for generating API docs
-$ sudo pip install sphinx pypandoc mkdocs
```

Comment:
So, to be more explicit, I think this seems installing bundled pandas properly in Mac or Windows given pypandoc's doc. Now, R documentation generation might not work in Mac or Windows.

Reply:
> bundled pandas

I assume you mean bundled pandoc.

I will add a note that pandoc is required and should be installed with whatever system package manager is appropriate. (Note that this is [what the Dockerfile does today](https://github.com/apache/spark/blob/c228810edcc2bc631029aa12a435aea701d061c2/dev/create-release/spark-rm/Dockerfile#L60-L61).)

Reply:
Sure, +1.